ontology/obcs.md

```diff
@@ -12,15 +12,21 @@ domain: information technology
 homepage: https://github.com/obcs/obcs
 in_foundry: false
 license:
-  label: CC BY 3.0
-  url: http://creativecommons.org/licenses/by/3.0/
+  label: CC BY 4.0
+  url: http://creativecommons.org/licenses/by/4.0/
 preferredPrefix: OBCS
 products:
 - id: obcs.owl
 repository: https://github.com/obcs/obcs
 tags:
 - statistics
 tracker: https://github.com/obcs/obcs/issues
+usages:
+- user: https://doi.org/10.1007/s40484-017-0122-5
+  description: The Ontology of Biological and Clinical Statistics (OBCS)-based statistical method standardization and meta-analysis of host responses to yellow fever vaccines
+  examples:
+  - url: https://doi.org/10.1007/s40484-017-0122-5
+    description: In Methods, "Both OBCS and the Vaccine Ontology (VO) were used to ontologically model various components and relations ..."
```

Comment:
@zhengj2007 I cant say anything against this because it is clearly in the OBO principles. But since you are OBO Ops I just want to mention that this kind of "usage evidence" is not really very robust. This just proves that OBCS has been used in a study at some point, not that it is "actively used". I will leave it to @deepakunni3 to use his authority to make a call here, but in my view this is:

1. Formally you have every right to get this approved, as it is stated in our principles
2. If we want to really use the `usages` field correctly, providing papers as evidence is not in the spirit of the inventor because everyone can write a word document explain how they used the ontology for NLP, upload it to Zenodo and record it here - which makes this field meaningless.

Reply:
@matentzn I know OBCS is used in [VEUPATH](https://veupathdb.org/). But I cannot find the link to the term since many ontology terms used in the  [VEUPATH](https://veupathdb.org/) annotations did not show the ontology terms directly. I can find some OBO ontologies import OBCS, such as [CIDO](https://ontobee.org/ontostat/catalog/CIDO?prefix=OBCS&iri=http://www.w3.org/2002/07/owl%23Class) and [EUPATH](https://ontobee.org/ontostat/catalog/EUPATH?prefix=OBCS&iri=http://www.w3.org/2002/07/owl%23Class)? Can I put the import as a usage?

I tried to add the usage section because it is a required field for passing the dashboard checking. Actually, I'd prefer to change it as warn rather than error for the first stage dashboard QC.

Reply:
Haha, you are talking to the wrong person. Sorry; I would like to be more strict here, as `usage` is a really important and hard-to-assess metric; imports are two easy again IMO to justify usage. You wont hear something better from me; I hear you want OBCS to pass the dashboard, and I wont stand in the way of that, as you are adhering to the verbatim of our rules - just saying I am not a fan of the way the rules are formulated.

If we set usage to "WARN", we basically decided that we don't care about this principle.. No one will ever make an effort to fill it in.

A better way IMO would be to find a way to prove that OBCS is used VEUPATH. Working with the VEUPATH people to update the pages and expose the OBCS IDs; even a data source documentation showing the use of OBCS would be ok.

Reply:
@matentzn I tried to find the OBCS terms in the VEUPATH. However, I left the VEUPATH team and cannot access the backend codes now. It is too hard to try to get the OBCS terms from over thousands of studies. The team cut whole ontology team. It's easy hard to ask them to document the ontologies used in the project. 

I don't say usage is not important. However, the ontology is hard to show strong usage evidences at beginning of development. But the developers have needs when they developed an ontology. What I mean for the first dashboard QC stage, I'd prefer to set it as WARN. Otherwise, most of new ontologies such as OccO will show error.

Reply:
I hear you! I just personally dont agree (neither with watering down usages to WARN, nor with allowing usage documentation without a life system and proof), but, the law is on your side, so you don't need my agreement! :)

Reply:
@deepakunni3 This debate has been finished in #2300 (see associated PRs), and I don't think we should rip this open again right now. From your end, you can only assess if @zhengj2007 adheres to this definition (which I think she does, but didn't dig deep).